tweetprocessors.js

```diff
@@ -113,13 +115,100 @@ class HashtagStats {
 
   /**
    * Returns a list of all the trending hashtags and countries.
+   * The list is ordered by country, hashtags with highest count.
+   * It is formattd as a list of [country, hashtag, entry].
    */
   get() {
-    // TODO: implement this
+    this.HashtagStats.findAll({order: ['country', 'count']}).then(full_list => {
+      var country_list = [];
+      for (var i = 0; i < full_list.length; i++) {
+        var country = full_list[i].dataValues.country;
+        var hashtag = full_list[i].dataValues.hashtag;
+        var count = full_list[i].dataValues.count;
+        var entry = Object.freeze([country, hashtag, count]);
+        country_list.push(entry);
+      }
+      return country_list.reverse();
+    });
```

Comment:
It looks like you're trying to return `country_list`, but this function is actually returning nothing, because it's an asynchronous operation. The `return` statement in the `then` block will return the result to the `then`, while the `get` function itself will return nothing. You should return the full promise instead (i.e. put a `return` before `this.HashtagStats.findAll`) so that the caller can use that.